packages/mobile/src/identity/contactMapping.ts

```diff
@@ -259,13 +257,19 @@ function* fetchWalletAddresses(e164Number: string) {
 
 // Returns a list of account addresses for the identifier received.
 export function* lookupAccountAddressesForIdentifier(id: string) {
+  const lostAccounts = yield call(lostAccountsChannel)
```

Comment:
nit: It's not a channel anymore, maybe rename to `fetchLostAccounts` or something?